src/styles/LearnTodayWord.scss

```
@@ -9,7 +9,7 @@
     font-style: normal;
 }
 
-.background {
+.learntodayword-background {
     display: flex;
     width: 100vw;
     padding: 80px 0px;
```

```suggestion
   padding: 80px 0;
```

Unit of measure 'px' is redundant.
보통 값이 0이라면 단위는 생략합니다.